### PR TITLE
Don't use environments in workflows

### PR DESCRIPTION
**Pull request by futurepaul**
these didn't help out with the problem of deploys from forks, and just make more busywork . so sad

### CODE REVIEW

**Review by benthecarman — APPROVED** [review]
lgtm

**Comment by TonyGiorgio** [comment]
Nice work LGTM

-------- Original Message --------

**Review by benthecarman — COMMENTED** [review]
I get this on the preview, I think this is from me not having created a node
image

File: frontend/src/routes/Settings.tsx [hunk 1]
```diff
@@ -152,6 +154,17 @@ function Settings() {
...
                         }
                     </div>
+                    {COMMIT_HASH &&
+                        <div>
+                            <p className="text-2xl font-light">Mutiny Version</p>
+                            <div className="flex items-center gap-4">
+                                <pre>
+                                    <code>{COMMIT_HASH}</code>
+                                </pre>
+                                <Copy copyValue={nodes[0]} />
```
Comment (benthecarman) [inline frontend/src/routes/Settings.tsx:164]:
```suggestion
                                <Copy copyValue={COMMIT_HASH} />
```

**Review by benthecarman — APPROVED** [review]
works for me!

**Comment by futurepaul** [comment]
yeehaw

**Pull request merged**